synapse/models/economic.py

```diff
@@ -209,17 +238,26 @@ def getModelDefs(self):
                     ('from:cash', ('bool', {}), {
                         'doc': 'Set to true if the payment input was in cash.'}),
 
+                    ('to:instrument', ('econ:pay:instrument', {}), {
```

Comment:
do we want to add a :type prop to distinguish payment types (e.g., credit, debit)

Reply:
That sounds like it's synonymous with the form linked in the `:from:instrument` and would be redundant.